### ISSUE TITLE
Increase sensitivity of travis

### ISSUE THREAD

**Issue by Karsten1987** [Karsten1987/roscon2018#32]
I've seen that travis still passes even though the current master hasn't been able to build correctly without warnings.
https://travis-ci.org/Karsten1987/roscon2018/builds/533585910

I was wondering if we can somehow change the granularity of error reporting to see that the current master comes with warnings.

**Comment by mikaelarguedas** [comment]
TL;DR I don't think there is a satisfying way of doing this.

colcon doesn't provide (that I know of) a way to return a different return code if stderr/warnings
IIRC ci.ros2.org uses a plug-in that parses the console output looking for warnings.


One issue is that if the action files generation create a bunch of CMake warnings related to connext so the job will always fail. We would need to have boilerplate to install connext, accept their licence, source connext setup file before every build and eat he extra build time, so no really a great solution.

We could register this repo on rosdistro to leverage the dev jobs and the warning reporting. But it will not be triggered nightly and will be based on the released packages and not the latest nightly build.
